di-ipv-core-stub/src/main/resources/templates/edit-user.mustache

```diff
@@ -184,6 +184,236 @@
 
             </div>
 
+            <div class="govuk-date-input" id="validFrom">
+
+                <div class="govuk-date-input__item">
+                    <div class="govuk-form-group">
+                        <label class="govuk-label govuk-date-input__label" for="validFrom-day">
```

Comment:
use camelCase or hyphenated identifiers for all id/name attribute values in this file
e.g. validFromDay 
or
valid-from-day

Reply:
I left it this format to be consistent with what is in for DoB but we can change this now to camel case and just highlight the difference when test pick it up